packages/dapp-high-roller/src/components/app-root/app-root.tsx

```diff
@@ -284,6 +296,8 @@ export class AppRoot {
           </CounterfactualTunnel.Provider>
         </main>
       </div>
+    ) : (
+      <h1 class="App message">connecting....</h1>
```

Comment:
Might be an opportunity for a better message here. "Loading state from state channels wallet" or "Connecting to wallet..."? Also separately we could re-use the loading animation with the 3 pulsing dots

Reply:
This is the same message TTT has. We can probably iterate it.